services/object/src/api.js

```diff
@@ -10,4 +11,25 @@ let builder = new APIBuilder({
   context: ['cfg', 'db', 'backends'],
 });
 
+builder.declare({
+  method: 'post',
+  route: '/upload/:name/:projectId',
```

Comment:
This will preclude object names containing `/`, which I think we will want.

Maybe move the `projectId` into the request body?